combinations/hash.tsv

```diff
@@ -273,4 +273,4 @@ ont-fast5-api=0.4.1,r-tailfindr=1.3.0
 gatk4=4.2.6.1,samtools=1.15.1,sambamba=0.8.2,bcftools=1.15.1,perl=5.32.1,r-base=4.1.2
 bwa=0.7.17,prinseq-plus-plus=1.2.4,perl=5.26
 levenshtein=0.20.1
-python=3.9.5,pysam=0.19.1,argparse=1.4.0,edlib=1.3.9,tqdm=4.64.0
+python,pysam,argparse,edlib,tqdm
```

Comment:
please leave the version in, but remove python.

Reply:
sorry, I was unsure about the error, thanks for the suggestion :)

Reply:
mh, argparse can also be removed